pkg/Directory.Build.props

```diff
@@ -1,21 +1,14 @@
 <Project>
-  <Import Project="..\Directory.Build.props" />
 
   <PropertyGroup>
-    <NoBuild>true</NoBuild>
-    <IncludeBuildOutput>false</IncludeBuildOutput>
-    <IncludeSymbols>false</IncludeSymbols>
-
-    <IsSymbolsPackage Condition="$(MSBuildProjectName.Contains('.symbols'))">true</IsSymbolsPackage>
+    <PackageAssetsPath>$(ArtifactsDir)pkgassets/</PackageAssetsPath>
+    <IncludeSymbols>true</IncludeSymbols>
```

Comment:
Are we still using the pkg folder and the nupkg projects? I thought they were going to be obsoleted.

Reply:
They are obsoleted. But I wanted to leave them in until we get the symbols packages working so that we can reference anything if we need to. Is that ok? Also, the `$(ArtifactsDir)pkgassets` is where the dll files go that are to be included in each package. Its under `Artifacts/pkgassets' and is a different folder then the `pkg` folder you are referring to.